deploy/examples/operator.yaml

```diff
@@ -514,6 +514,8 @@ data:
 
   # Enable topology based provisioning.
   CSI_ENABLE_TOPOLOGY: "false"
+  # RevisionHistoryLimit value for all deployments created by rook.
+  # ROOK_REVISION_HISTORY_LIMIT: "3"
```

Comment:
Please move this new setting down to the end of the configmap around line 570. Currently, it's in the middle of the csi settings. (same for operator-openshift.yaml)

Reply:
done